### ISSUE TITLE
A lot of temporary (compression?) folders in /backup

### ISSUE THREAD

**Issue by n1trux** [serghey-rodin/vesta#1035]
### Operating System (OS/VERSION):

Debian 8

### VestaCP Version:

0.9.8-17

### Installed Software (what you got with the installer):

php-fpm, nginx, mysql

### Steps to Reproduce:

In `/backup` there are a lot of folders with names like this:

    /backup# ls

    tmp.02XNOVzMc3	  tmp.cG04uNVBlw  tmp.KaEJL1L5Ij  tmp.UmHTmBc9Sb
    tmp.0LkazdQQnI	  tmp.CIjmlNkyeR  tmp.keg59CHpvK  tmp.UNfi2W5fCo
    tmp.0rFJZiOpZs	  tmp.cOtGEdn7zn  tmp.KGD3OBFoJp  tmp.unIBsFkAPW
    tmp.0V94gaLKwY	  tmp.Cr3smmyCLD  tmp.KHoIvJsZUG  tmp.UqphURqEu8

Inside of them are user directories, I suspect they're leftovers from compressing. How do we fix this?

**Comment by SCelik** [comment]
Are you sure that your backup jobs compated without any errors? Those files should be removed after backup job automaticly.

**Comment by n1trux** [comment]
Yeah, in fact the files which are in the tmp.xxx folders are also in current backups…

**Comment by SCelik** [comment]
Just delete them, and see what will happen next run. I'm sure that there should be an uncompleted job before.

**Comment by Skamasle** [comment]
Sometimes you can get this when you run out of space

**Comment by n1trux** [comment]
119GB used, 220GB free :/

No errors in `/var/log/vesta/backup.log`, runtime is always under ten minutes.

**Comment by stathis** [comment]
Been having the same issue. It's okay to delete them, but it's irritating.

**Comment by Skamasle** [comment]
What is the serverload ?

If you run on high load backup can wait if load never down it maybe idle

I see this tmp problem in some server but not have time to investigate it yet

**Comment by FalzoMAD** [comment]
possible cause and solution described here: https://forum.vestacp.com/viewtopic.php?f=41&t=13286

**Comment by 001101** [comment]
had also the issue, hope the patch will fix it, have eaten 100gb space in about 2 hours!!
